### PR TITLE
Calendar bug fixes

### PR DESCRIPTION
**Pull request by juanlaube**
## What's this?
This includes some bug fixes on iOS for the calendar view after the first test build.

## Why do we want this?
🚫 🐛 NO BUGS ALLOWED

## How is it done?
This fixes:
- Swiping to future dates in the calendar view
- The current time indicator wasn't displaying for the current date
- A glitch where the running time entry card appeared on top of the contextual menu

### Side effects
None

## :squid: Permissions
🦑

### CODE REVIEW

File: Toggl.Core.Tests/UI/ViewModels/CalendarViewModelTests.cs [hunk 1]
```diff
@@ -20,6 +20,7 @@
 using Xunit;
 using Toggl.Core.UI.Helper;
 using System.Globalization;
+using Microsoft.VisualStudio.TestPlatform.CrossPlatEngine;
```
Comment (simonrozsival) [inline Toggl.Core.Tests/UI/ViewModels/CalendarViewModelTests.cs:23]: 👀

Comment (juanlaube) [reply Toggl.Core.Tests/UI/ViewModels/CalendarViewModelTests.cs:23]: 🙈

**Comment by juanlaube** [comment]
@sofokli1 yes!

**Comment by juanlaube** [comment]
@sofokli1 yeah there's an API limitation, the date range could be bigger, but we still need to set a limit somewhere. Re not being able to go into the future, try to uninstall, clean, rebuild and install again, that worked for Simon.

**Pull request merged**